app/src/components/GHGIHomePage/DownloadAndShareModals/PublishButtons.tsx

```diff
@@ -43,7 +43,8 @@ const PublishButtons = ({
         <Button
           key={title}
           my="16px"
-          mx="16px"
+          pr="16px"
+          pl="35px"
```

Comment:
Why does the padding need to be different on the left and right side of the button?